packages/playwright/src/reporter.ts

```diff
@@ -378,13 +390,16 @@ export default class ReplayPlaywrightReporter implements Reporter {
       }
 
       if (output.length) {
-        output.push("");
         output.push(
           `Learn more at ${link(
             "https://docs.replay.io/reference/test-runners/playwright/overview"
           )}`
         );
-        output.forEach(line => {
+
+        output.forEach((line, index) => {
+          if (index > 0) {
+            console.log("[replay.io]:");
+          }
```

Comment:
Not a concern right now but I feel it will make it slightly harder to create multiline logs that should not have an empty line between them.

Reply:
Yeah. I don't love the way we are doing logging here. I just kept the previous behavior (minus the occasional awkward trailing log).

Reply:
I dunno maybe this is better

```
  2 passed (9.0s)
[replay.io]: Finishing up. This should only take a moment ...
[replay.io]: 
[replay.io]: 🚀 Successfully uploaded 1 recordings:
[replay.io]: 
[replay.io]:    ✅ login-logout-library: should login and logout from the library
[replay.io]:       https://app.replay.io/recording/8a18b945-7842-4505-b8aa-c4537c28b908
```